web/core/store/project/project.store.ts

```
@@ -214,23 +224,22 @@ export class ProjectStore implements IProjectStore {
   };
 
-  getProjectEpicPropertiesById = computedFn((projectId: string | undefined | null) => {
-    const projectEpicProperties = this.projectEpicPropertiesMap[projectId ?? ""];
-    return projectEpicProperties;
-  });
+  setLastCollapsibleAction = (section: ProjectOverviewCollapsible) => {
+    this.openCollapsibleSection = [section];
+  };
+
+  toggleOpenCollapsibleSection = (section: ProjectOverviewCollapsible) => {
+    if (this.openCollapsibleSection && this.openCollapsibleSection.includes(section)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)